### PR TITLE
[DNM] Non-standard Performance Logging

### PR DESCRIPTION
**Pull request by Gandalf2k15**
Do not merge this. 

Performance logging for individual subsystems.

### CODE REVIEW

**Review by GoldenAlpharex — APPROVED** [review]
Should be good for a test-merge.

File: code/controllers/subsystem/time_track.dm [hunk 1]
```diff
@@ -59,23 +70,10 @@ SUBSYSTEM_DEF(time_track)
 			"tidi_slowavg",
 			"maptick",
 			"num_timers",
-			"air_turf_cost",
-			"air_eg_cost",
-			"air_highpressure_cost",
-			"air_hotspots_cost",
-			"air_superconductivity_cost",
```
Comment (Iamgoofball) [inline code/controllers/subsystem/time_track.dm:66]: why are you removing performance logging?

File: code/controllers/subsystem/time_track.dm [hunk 2]
```diff
@@ -59,23 +70,10 @@ SUBSYSTEM_DEF(time_track)
...
-			"air_turf_count",
-			"air_eg_count",
-			"air_hotspot_count",
-			"air_network_count",
-			"air_delta_count",
-			"air_superconductive_count",
 			"all_queries",
 			"queries_active",
 			"queries_standby"
-		) + sendmaps_headers
```
Comment (Iamgoofball) [inline code/controllers/subsystem/time_track.dm:78]: and why are you removing the sendmaps logging?

Comment (Gandalf2k15) [reply code/controllers/subsystem/time_track.dm:78]: To keep the csv file from killing Google sheets, the lag isn't related to sendmaps, plus this is just to gather other metrics

**Pull request closed without merging**